gangplank/ocp/bc.go

```diff
@@ -202,12 +206,52 @@ binary build interface.`)
 		return err
 	}
 
+	if err := m.ensureBucketExists(ctx, "binary"); err != nil {
+		return err
+	}
+	defer m.deleteBucket(ctx, "binary")
+
+	if bc.SrvDir != "" {
```

Comment:
https://github.com/coreos/coreos-assembler/pull/2059/files#diff-651847a17849b0f40186f5dfde966683921fb490d8b75e5797ff3a1bdc120aecR310 is the result of my thinking about this problem -- for publication stages we could just set `RequiredArtifacts` on the stage. I did it for _building_, but I think that this would be useful here.